From: Baruch Siach <baruch@tkos.co.il>
Date: Tue, 27 Oct 2015 14:23:44 +0200
Subject: [PATCH] lib/Makefile.am: fix lanplus disable

Protect the install target as well when lanplus is disabled. Fixes the
following installation failure when openssl is missing:

In file included from lanplus.c:78:0:
./inc/ipmitool/ipmi.h:51:25: fatal error: openssl/evp.h: No such file or directory

Patch status: sent upstream
(http://sourceforge.net/p/ipmiutil/mailman/message/34572580/)

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 lib/Makefile.am | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 805a218ab7eb..939594cfd3e5 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -51,8 +51,10 @@ distclean:
 	cd lanplus; make distclean
 	
 install:
-	$(MKDIR) ${datato}
-	cd lanplus; make install
+	if [ "$(PLUSFLAGS)" = "-DHAVE_LANPLUS" ]; then \
+	  $(MKDIR) ${datato} ; \
+	  cd lanplus; make install ; \
+	fi
 
 check:
 
-- 
2.6.1

